examples/basic/client/client.iml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4" />
```

Comment:
I think we should keep idea things out of the repo.

Reply:
Correct. I missed this one. I will exclude it.